### ISSUE TITLE
Deploying from Editor uses .bim model name as model name - changing model name from project file

### ISSUE THREAD

**Issue by C45P3R** [otykier/TabularEditor#43]
As the title says. When you use Tabular Editor to deploy it uses the model name from the .bim file. 
Which is problematic in cases where that name doesn't match the model name otherwise used. EI deploying fra VS using the model name found in the project file.

Suggestions: let the user select a project file and get the name from there or let the user fill in a textbox with model name in the deploy process.

**Comment by otykier** [comment]
Hi C45P3R. Is there a reason you don't want to change the Model name that appears in the Model.bim file? Tabular Editor exposes this field as the "Name" property on the root model object:

![image](https://cloud.githubusercontent.com/assets/8976200/25249321/ce452a9c-2611-11e7-9546-649d0da9e5e1.png)

**Comment by otykier** [comment]
One additional note - if you want to apply a change of model name only during deployment, you can use a new feature of Tabular Editor's command-line deployment. In release 2.3, you can supply a script when deploying from the command-line.

C:\Projects\AdventureWorks\MyScript.cs:

```C#
Model.Name = "Model Test Changed Name";
```

When deploying, use the following command-line syntax:

`start /wait TabularEditor.exe "c:\Projects\AdventureWorks\Model.bim" -S "c:\Projects\AdventureWorks\MyScript.cs" -D localhost AdventureWorks -O`

Basically, this loads the Model.bim file into Tabular Editor, applies the script specified after the "-S" switch, and then deploys ("-D" switch) to the localhost server overwriting ("-O" switch) the AdventureWorks database.

For a full list of command-line syntax, run: `TabularEditor.exe /?`

**Comment by otykier** [comment]
Closing this. Feel free to reopen if you have any further questions.
